cmd/podman/pods/create.go

```diff
@@ -62,6 +62,10 @@ func init() {
 	flags := createCommand.Flags()
 	flags.SetInterspersed(false)
 	common.DefineCreateDefaults(&infraOptions)
+	// these settings are not applicable to pod create since they are per-container
+	// and they will end up being duplicated for each container in the pod.
+	infraOptions.Volume = nil
+	infraOptions.Mount = nil
```

Comment:
Barely on the fence here.  Should we add a debug statement just above this, showing the values we're wiping?  I think not, but I don't know for sure if that might be useful at some far-flung point in time.

Reply:
not sure, I can add that if you think it is useful